src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/HtmlCSharp/SignatureHelpHandlerTest.cs

```diff
@@ -0,0 +1,217 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Microsoft.VisualStudio.Threading;
+using Moq;
+using Xunit;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor.HtmlCSharp
+{
+    public class SignatureHelpHandlerTest
+    {
+        public SignatureHelpHandlerTest()
+        {
+            Uri = new Uri("C:/path/to/file.razor");
+
+            SignatureHelp = new SignatureHelp()
```

Comment:
Wrote this out with the intention of testing against the contents, but decided against it in the end considering it's just data fed to us from VS LSP / roslyn -> didn't think it would be too beneficial to specifically unit test that. Keeping it in should it be needed in the future.

Reply:
Suggest removing it if it isn't currently being used. We can always re-add it from the git history.